service/src/main/java/edu/tamu/catalog/service/FolioCatalogService.java

```diff
@@ -693,8 +693,9 @@ private boolean getManualBlockStatus(String userId) {
             JsonNode okapiHoldings = getOkapiHoldings(instanceId);
             okapiHoldings.forEach(holding -> {
                 String hrid = holding.at("/hrid").asText();
-                String fallbackLocationCode = getLocation(holding.at("/permanentLocationId").asText()).at("/code").asText();
-
+                JsonNode holdingLocationNode = getLocation(holding.at("/permanentLocationId").asText());
```

Comment:
You will want to use getText for these to avoid null pointer exception.
https://github.com/TAMULib/CatalogService/blob/64f97e99035c1616f03d3da4e856cbdd7505a94b/service/src/main/java/edu/tamu/catalog/utility/JsonNodeUtility.java#L20